### PR TITLE
Vet-Clinic

### PR DESCRIPTION
**Pull request by theodoku**
In this PR:

- [ ] Create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
screenshot

- [ ] Insert the following data:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
screenshot

- [ ] Find all animals whose name ends in "mon".
screenshot

- [ ] List the name of all animals born between 2016 and 2019
screenshot

- [ ] List the name of all animals that are neutered and have less than 3 escape attempts
screenshot

- [ ] List the date of birth of all animals named either "Agumon" or "Pikachu"
screenshot

- [ ] List name and escape attempts of animals that weigh more than 10.5kg
screenshot

- [ ] Find all animals that are neutered
screenshot

- [ ] Find all animals not named Gabumon
screenshot

- [ ] Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +131,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (AlphaNtihinduka) [inline README.md:132]: - [x] Kindly ensure that you add the `MIT`. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @theodoku 🙋‍♂️
Good job working on creating the animal table. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- Kindly consider the inline review comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 1 above)
Comment (theodoku) [reply README.md:132]: MIT License added👍

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @theodoku 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**